components/standings/commons/standings_storage.lua

```diff
@@ -23,6 +23,37 @@ local ALLOWED_SCORE_BOARD_KEYS = {'w', 'd', 'l'}
 local SCOREBOARD_FALLBACK = {w = 0, d = 0, l = 0}
 local DISQUALIFIED = 'dq'
 
+---@class StandingsTableStorage
+---@field standingsindex integer
+---@field title string?
+---@field type 'ffa'|'swiss'|'league'
```

Comment:
[oos] would we eventually want to differentiate roundrobin, doubleroundrobin from other league ones?
(others would be non exclusive groups, gsl style, basically any non swiss non (double) roundrobin)

Reply:
In the future maybe, but only if there's a strong enough reason. I'm not sure I see a reason for diffing RR and DRR for example at this level (that's more of minor format change)